ui/pages/DataManagement/components/ElasticsearchStatus.jsx

```diff
@@ -35,7 +35,11 @@ const INDEX_COLUMNS = [
       </div>
     )) : <DeleteIndexButton index={row.index} />),
   },
-  { name: 'datasetType', content: 'Caller Type', format: row => (row.datasetType === 'SV' ? 'SV' : 'SNV') },
+  {
+    name: 'datasetType',
+    content: 'Caller Type',
+    format: row => (row.datasetType === 'SV' || row.datasetType === 'MITO' ? row.datasetType : 'SNV'),
```

Comment:
I think it would be neater of we reversed the direction of the check, and also we really should use constants and not hard code strings for the dataset types - 
```
row.datasetType === DATASET_TYPE_VARIANT_CALLS ? 'SNV' : row.datasetType
```